File: Programmin Fundamentals/Homeworks and Labs/FilesAndExceptionsExercises/02.IndexOfLetters/IndexOfLetters.cs
namespace _02.IndexOfLetters
{
    using System;
    using System.IO;
    using System.Linq;

    public class IndexOfLetters
    {
        public static void Main()
        {
            var alphabet = "abcdefjhijklmnopqrstuvwxyz".ToCharArray();
            var letters = Console.ReadLine().ToCharArray().Select(x => x.ToString()).ToArray();

            File.WriteAllLines("inputNumbers.txt", letters);
            File.WriteAllText("output.txt", string.Empty);

            foreach (var letter in letters)
            {
                File.AppendAllText("output.txt", $"{letter} -> {Array.IndexOf(alphabet, char.Parse(letter))}{Environment.NewLine}");
            }
        }
    }
}

File: Software Technologies/Java Basics - Exercises/src/IndexOfLetters.java
import java.util.Scanner;

public class IndexOfLetters {
    public static void main(String [] args)
    {
        Scanner scanner = new Scanner(System.in);

        char [] letters = scanner.nextLine().toCharArray();

        for (int i = 0; i < letters.length; i++) {
            char letter = letters[i];
            System.out.printf("%s -> %d",letter,letter-97);
            System.out.println();
        }

    }
}
